config/dump.c

```diff
@@ -139,7 +139,28 @@ void dump_config_neo(struct ConfigSet *cs, struct HashElem *he, struct Buffer *v
   if (show_name && show_value)
     fprintf(fp, "set ");
   if (show_name)
-    fprintf(fp, "%s", name);
+  {
+    if (flags & CS_DUMP_LINK_DOCS)
+    {
+      // Used to generate unique ids for the urls
+      static int seq_num = 1;
+      static const char *url = "https://neomutt.org/guide/reference";
+      char fragment[128] = { 0 };
```

Comment:
How about mutt_str_dup instead? I'm sure 128 is big enough for our config names, but `my` variables could be longer.

Reply:
ah good point.
I also need a different url for myvars.
thanks, done